<?xml version="1.0"?>

<launch>

	<rosparam file="$(find drones)/config/params.yaml" command="load"/>

	<group ns="uav1">
	    <node name="drone_operator" pkg="drones" type="drone_operator" output="screen">
	        <param name="uav_id" type="int" value="1" />
	        <remap from="/gazebo/model_states" to="/gazebo/model_states_fake"/>
	    </node>
	</group>

	<group ns="uav2">
	    <node name="drone_operator" pkg="drones" type="drone_operator" output="screen">
	        <param name="uav_id" type="int" value="2" />
	        <remap from="/gazebo/model_states" to="/gazebo/model_states_fake"/>
	    </node>
	</group>

	<group ns="uav3">
	    <node name="drone_operator" pkg="drones" type="drone_operator" output="screen">
	        <param name="uav_id" type="int" value="3" />
	        <remap from="/gazebo/model_states" to="/gazebo/model_states_fake"/>
	    </node>
	</group>

</launch>

